app/(Base)/results/page.tsx

```diff
@@ -1,38 +1,87 @@
 'use client';
 import React from 'react';
-import Link from 'next/link';
+import { StatCard } from '@/components/StatCard';
+import {
+   Select,
+   SelectContent,
+   SelectItem,
+   SelectTrigger,
+   SelectValue,
+} from '@/components/UI/select';
+import { accurateWPM, formatTime, grossWPM } from '@/helpers/utils/util';
 import { userConfigStore, wordStore } from '@/store';
-import { accurateWPM, grossWPM } from '@/helpers/utils/util';
+import Link from 'next/link';
+
+const LANGUAGES = ['English', 'Spanish', 'French', 'German'];
+const SHOW_ALL_TIME_STATS = false;
 
 export default function ResultsPage() {
    const { typedEntries, errorCount } = wordStore.getState();
    const { time, category } = userConfigStore.getState();
 
-   const rawWpm = grossWPM(typedEntries, time);
+   const rawWpmw = grossWPM(typedEntries, time);
```

Comment:
why rename the variable by just adding the 'w'.. does it mean anything? 😆 😆 😆

Reply:
i was the one that did that?? I honestly didn't know 🤣 . I was surprised when I saw the name changed, I just thought that was how it was and left it like that. 

Should I push a commit removing it?

Reply:
I will deal with it..dw..
nice job @d-a-ve 👍🏽 
Really appreciate. 🙌🏽